client/src/Game.h

```diff
@@ -12,14 +12,18 @@
 #include "NetworkThread.h"
 #include <Tilemap.h>
 #include <QuadTree.h>
+#include <EntityManager.h>
 
 
 class Game {
     sf::RenderWindow mWindow;
-    Player player;
-    PlayerCommands playerCommands;
+
+    EntityManager *entityManager;
```

Comment:
Same as keeping a pointer to the player, it's more convenient. But as you wish ?